nx-completion.plugin.zsh

```diff
@@ -330,20 +343,22 @@ _nx_command() {
     (run-many)
       _arguments $(_nx_arguments) \
         $opts_help \
-        "--version[Show version number.]" \
-        "--target[Task to run for affected projects.]:target:" \
-        "--parallel[Parallelize the command.]" \
-        "--maxParallel[Max number of parallel processes.]:count:" \
-        "--projects[Projects to run (comma delimited).]:projects:_list_projects" \
-        "--all[Run the target on all projects in the workspace.]" \
+        "--all[(deprecated) Run the target on all projects in the workspace]" \
+        "--configuration[This is the configuration to use when performing tasks on projects]:configuration:" \
+        "--exclude[Exclude certain projects from being processed]:projects:_list_projects:" \
+        "--nx-bail[Stop command execution after the first failed task]" \
+        "--nx-ignore-cycles[Ignore cycles in the task graph]" \
+        "--output-style[Defines how Nx emits outputs tasks logs]:style:(dynamic static stream stream-without-prefixes)" \
+        "(--parallel --maxParallel)"{--parallel,--maxParallel}"[Max number of parallel processes. (default is 3)]:count:" \
+        "--projects[Projects to run (comma delimited).]:projects:_nx_list_projects" \
         "--runner[Override the tasks runner in nx.json.]:runner:" \
-        "--skipNxCache[Rerun the tasks even when the results are available in the cache.]" \
-        "--configuration[This is the configuration to use when performing tasks on projects.]:configuration:" \
-        "--withDeps[TInclude dependencies of specified projects when computing what to run.]" \
-        "--onlyFailed[Isolate projects which previously failed.]" \
-        "--verbose[Print additional error stack trace on failure.]" && ret=0
+        "(--skip-nx-cache --skipNxCache)"{--skip-nx-cache,--skipNxCache}"[Rerun the tasks even when the results are available in the cache.]" \
+        "(-t --target --targets)"{-t=,--target=,--targets=}"[Task(s) to run for affected projects.]:target:" \
+        "--version[Show version number.]" \
+        "--verbose[Print additional error stack trace on failure.]" \
+        && ret=0
     ;;
-    (run)
+    (run|run-one)
```

Comment:
interestingly, this alias was removed from nx 15.7 -> 15.8 >_<

Reply:
Weird, this is not documented in their release notes.